app/config/config.yml

```diff
@@ -10,7 +10,7 @@ parameters:
 
 framework:
     #esi:             ~
-    #translator:      { fallbacks: ["%locale%"] }
+    translator: ~
```

Comment:
I was just following the instruction in symfony documentation, but should i put the { fallbacks: ["%locale%"] } so later we would be able to use the tigrigna language?